src/terraform/index.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // terraformModulesCommitHash is the git commit hash that reflects
 // which of the Terraform modules in Originate/exosphere we are using
-const terraformModulesCommitHash = "8786f912"
+const terraformModulesCommitHash = "48731e6"
```

Comment:
It doesn't have to be the latest commit, but it should be the latest commit that changed anything in `terraform/modules`. What would that automatic update look like?